src/servers/src/grpc/frontend_grpc_handler.rs

```diff
@@ -41,12 +43,27 @@ impl Frontend for FrontendGrpcHandler {
         } else {
             Some(list_process_request.catalog.as_str())
         };
-        match self.process_manager.local_processes(catalog) {
-            Ok(processes) => Ok(Response::new(ListProcessResponse { processes })),
-            Err(e) => {
-                error!(e; "Failed to handle list process request");
-                Err(Status::new(Code::Internal, e.to_string()))
-            }
-        }
+        let processes = self.process_manager.local_processes(catalog).map_err(|e| {
+            error!(e; "Failed to handle list process request");
+            Status::new(Code::Internal, e.to_string())
+        })?;
+        Ok(Response::new(ListProcessResponse { processes }))
```

Comment:
We only support listing processes in the current frontend?

Reply:
Process list info is collected from all frontend nodes by sending `ListProcessRequest`:
https://github.com/GreptimeTeam/greptimedb/blob/05b708ed2e241eb47e26340a69ca86b571780f4a/src/catalog/src/system_schema/information_schema/process_list.rs#L118-L126

This method is for frontend to answer the `ListProcessRequest` rpc request.